### PR TITLE
Fixed yamllint warning and errors

### PR DESCRIPTION
**Pull request by pewo**
Added a variable to fix yamllint error    `line too long (87 > 80 characters)  (line-length)`

```yaml
thinlinc_pack: "{{ thinlinc_version }}-{{ thinlinc_build }}"
```
And used that variable in all vars files

Fixed some other minor warnings  ` truthy value should be one of [false, true]  (truthy)`
```yaml
changed_when: False
changed_when: false
```
And some other warnings and errors:
`error    too many blank lines (1 > 0)  (empty-lines)`
`error    no new line character at the end of file  (new-line-at-end-of-file)`
`warning  missing document start "---"  (document-start)`

With these fixes yamllint reports nothing

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ thinlinc_accept_eula: "no"
 thinlinc_version: "4.12.1"
 thinlinc_build: "6733"
 thinlinc_server_bundle: "tl-4.12.1-server.zip"
+thinlinc_pack: "{{ thinlinc_version }}-{{ thinlinc_build }}"
```
Comment (CendioOssman) [inline defaults/main.yml]: Sorry for being unclear, but the goal was one commit per type of change. Not per file. So all of these would be in the same commit.

That said, I think these are just making it difficult to read. So I think we should disable this lint rule, at least for the constants that define the file names.

File: defaults/main.yml [hunk 2]
```diff
@@ -14,7 +15,10 @@ thinlinc_printers: "yes"
 
 # ThinLinc Web Administration password. This default password is
 # "thinlinc". Generate new hashes with /opt/thinlinc/sbin/tl-gen-auth.
-thinlinc_webadm_password: "$6$7cc31a35e02e55ec$hm.1MsloeBJqNKljx9RH88Z/eRKZCka5ZlabkZGj0nYXh0IaxaiYucsDD.fGJ5sNPthWf63pXkCn9Nu0ua2Ye1"
+thinlinc_webadm_password: "\
+    $6$7cc31a35e02e55ec$hm.1MsloeBJqNKljx9RH88Z/eRKZCka5Z\
+    labkZGj0nYXh0IaxaiYucsDD.fGJ5sNPthWf63pXkCn9Nu0ua2Ye1\
+"
```
Comment (CendioOssman) [inline defaults/main.yml]: Same thing here. This just makes it difficult to copy and paste things.

**Review by CendioOssman — CHANGES_REQUESTED** [review]

**Comment by pewo** [comment]
Restarted from the 1.6 release,  added .yamllint file and fixed small things like newline/blank lines/---.
yamllint does not report any warnings or errors.

**Comment by CendioOssman** [comment]
Great, thanks.

**Pull request merged**